### PR TITLE
remove message hash and use validation message from sprint

### PR DESCRIPTION
**Pull request by dougperez**
For validate device v1 we needed to translate the message, this is no
longer with sprint version 2.

### CODE REVIEW

File: lib/conduit/sprint/parsers/query_device_info.rb [hunk 1]
```diff
@@ -85,18 +68,10 @@ class QueryDeviceInfo::Parser < Parser::Base
       content_for '//uiccNotAvailableReasonCode'
     end
 
-    attribute :uicc_not_available_reason_message do
```
Comment (pedromartinez) [inline lib/conduit/sprint/parsers/query_device_info.rb]: Don't we have to replace this with the API response field?

Comment (dougperez) [inline lib/conduit/sprint/parsers/query_device_info.rb]: For sprint there is only one :validation_message field.

Comment (pedromartinez) [inline lib/conduit/sprint/parsers/query_device_info.rb]: Wait, what?

Comment (dougperez) [inline lib/conduit/sprint/parsers/query_device_info.rb]: Yeah they send back multiple codes that we already capturing. This was just a message field we were translating which the ```validation_message``` already returns for either type.

Comment (dougperez) [inline lib/conduit/sprint/parsers/query_device_info.rb]: @pedromartinez anything we need to change here?

Comment (pedromartinez) [inline lib/conduit/sprint/parsers/query_device_info.rb]: No, seems fine for now.

**Pull request merged**